README.md

```diff
@@ -404,6 +404,17 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
         },
 
         // Defines options for the connection pool.
+        //
+        // The options defined in this section are only used when
+        // "enable_4_2_compatibility" is set to false.
+        //
+        // When connection pooling is used, the HTTP API will not be able
+        // to detect changes in policy within the connected iRODS server.
+        // Any changes in policy will require a restart of the HTTP API.
+        //
+        // If this is a concern, set "enable_4_2_compatibility" to true to
+        // force the HTTP API to use a new iRODS connection for every HTTP
+        // request.
```

Comment:
Perhaps if there is already a mention about the performance hit in the 4.2. compatibility section vis a vis no connection pooling, we can refer to it here or just use the same wording. In other words, having it in both places one way or the other seems good to me.

Reply:
I thought I added words somewhere about that. Will double check.

Reply:
i'm not sure we need MORE words other than what's above...
'force the HTTP API to use a new iRODS connection for every HTTP request'.

i mean, that doesn't sound 'fast'.   and we're not going to publish actual metrics / numbers, i would think.

Reply:
I can see it going either way.

The change in performance is totally implied and that may be good enough. However, that means we're leaving it up to the reader to assume things about the application behavior. Because of that, I'm leaning in the direction of adding a few words stating performance is degraded when 4.2 compat is enabled. That removes any room for users assuming the wrong thing.

Thoughts?

Reply:
that's fair... perhaps an additional sentence of...

`The additional connections will honor any iRODS server policy changes, but will degrade overall performance.`

Reply:
Will give that a try.

Reply:
Done. Let me know what you think.